pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller_test.go

```diff
@@ -41,6 +41,9 @@ func newClusterManager(name string) *operatorapiv1.ClusterManager {
 		},
 		Spec: operatorapiv1.ClusterManagerSpec{
 			RegistrationImagePullSpec: "testregistration",
+			DeployOption: operatorapiv1.DeployOption{
+				Mode: operatorapiv1.InstallModeDefault,
+			},
```

Comment:
I think it needs another test for detached mode.

Reply:
Yes, but for another PR, in this PR the detached functionality is not included. In this PR we will use an e2e test to make sure detached cluster-manager can be installed in a customize namespace and that's all.

Reply:
Solved, a relevant e2e test case is ongoing.